tendrl/node_agent/manager/pull_hardware_inventory.py

```diff
@@ -28,7 +28,7 @@ def getNodeCpu():
                    "CoresPerSocket": "corespersocket"}, ...], ...}
 
     '''
-    cmd = Command({"_raw_params": "lscpu"})
+    cmd = cmd_utils.Command({"_raw_params": "lscpu"})
     out, err = cmd.start()
```

Comment:
stdout, stderr, rc = cmd.run() would be the new way to invoke this. Can you please update this